### ISSUE TITLE
Bug that gets me into the login screen of my linux system

### ISSUE THREAD

**Issue by MalekDeKalem** [pystardust/ani-cli#1129]
**Metadata (please complete the following information)**
Version: 4.2.0
OS: Garuda KDE (based on arch linux)
Shell: fish
Anime: kono bijutsu

Description:
So I basically execute the ani-cli script and choose my anime like normally.
However a few seconds after I choose an episode my screen goes black and
I get teleported into the login page of my linux system, where I then have to type in 
my root password. The fans on my PC also go haywire as if they were A-10 fighter jets, which terrifies me.
I just wanted to watch my relaxing "slice of life" shows instead I got fear instilled within me.

I am sorry that I could not provide any error logs.

**Steps To Reproduce**

1. Run `ani-cli kono bijutsu`
2. Choose episode 12

**Expected behavior**
Watch Anime

**Comment by Sarvasv-0** [comment]
> **Metadata (please complete the following information)** Version: 4.2.0 OS: Garuda KDE (based on arch linux) Shell: fish Anime: kono bijutsu
> 
> Description: So I basically execute the ani-cli script and choose my anime like normally. However a few seconds after I choose an episode my screen goes black and I get teleported into the login page of my linux system, where I then have to type in my root password. The fans on my PC also go haywire as if they were A-10 fighter jets, which terrifies me. I just wanted to watch my relaxing "slice of life" shows instead I got fear instilled within me.
> 
> I am sorry that I could not provide any error logs.
> 
> **Steps To Reproduce**
> 
> 1. Run `ani-cli kono bijutsu`
> 2. Choose episode 12
> 
> **Expected behavior** Watch Anime

is this the newly updated version?

**Comment by Derisis13** [comment]
Look at your system resource usage. It sounds like an issue outside ani-cli, that crashes/kills KDE when trying to play a video. You can also try using vlc as the player, see if that makes it different (refer to man page on how)

**Comment by port19x** [comment]
Unlikely to be a direct issue with ani-cli, but we can give you support anyway.
I agree with what @Derisis13 says.
Additionally, `sudo journalctl -b 0` gives you the system log of the current boot, while `sudo journalctl -b -1` gives you the previous one, if you reboot.
Please let us know what it was, this is intriguing

**Comment by MalekDeKalem** [comment]
Thanks for being so helpful about, it I am using now the 4.3.0 Version of the script. But my issue still isn't resolved I also ran `ani-cli kono bijutsu` and chose the 12th episode (reproducing my issue) and then after I logged in again opened up a termnial and typed `sudo journalctl -b 0` to get the system log of this boot.
Here is my log:
Mai 17 14:07:15 dorian-ms7a38 kernel: Linux version 6.2.2-zen2-1-zen (linux-zen@archlinux) (gcc (GCC) 12.2.1 20230201, GNU ld (GNU Binutils) 2.40) #>
Mai 17 14:07:15 dorian-ms7a38 kernel: Command line: BOOT_IMAGE=/@/boot/vmlinuz-linux-zen root=UUID=a608f7f2-5ed0-419c-8dc1-3f770dc5473a rw rootflags>
Mai 17 14:07:15 dorian-ms7a38 kernel: x86/fpu: Supporting XSAVE feature 0x001: 'x87 floating point registers'
Mai 17 14:07:15 dorian-ms7a38 kernel: x86/fpu: Supporting XSAVE feature 0x002: 'SSE registers'
Mai 17 14:07:15 dorian-ms7a38 kernel: x86/fpu: Supporting XSAVE feature 0x004: 'AVX registers'
Mai 17 14:07:15 dorian-ms7a38 kernel: x86/fpu: xstate_offset[2]:  576, xstate_sizes[2]:  256
Mai 17 14:07:15 dorian-ms7a38 kernel: x86/fpu: Enabled xstate features 0x7, context size is 832 bytes, using 'compacted' format.
Mai 17 14:07:15 dorian-ms7a38 kernel: signal: max sigframe size: 1776
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-provided physical RAM map:
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x0000000000000000-0x000000000009d3ff] usable
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x000000000009d400-0x000000000009ffff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000000e0000-0x00000000000fffff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x0000000000100000-0x0000000003ffffff] usable
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x0000000004000000-0x000000000400ffff] ACPI NVS
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x0000000004010000-0x0000000009d7ffff] usable
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x0000000009d80000-0x0000000009ffffff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x000000000a000000-0x00000000dd521fff] usable
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000dd522000-0x00000000dd661fff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000dd662000-0x00000000dd75cfff] usable
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000dd75d000-0x00000000ddb2dfff] ACPI NVS
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000ddb2e000-0x00000000de667fff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000de668000-0x00000000deffffff] usable
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000df000000-0x00000000dfffffff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000f8000000-0x00000000fbffffff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000fea00000-0x00000000fea0ffff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000feb80000-0x00000000fec01fff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000fec10000-0x00000000fec10fff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000fec30000-0x00000000fec30fff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000fed00000-0x00000000fed00fff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000fed40000-0x00000000fed44fff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000fed80000-0x00000000fed8ffff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000fee00000-0x00000000feefffff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x00000000ff000000-0x00000000ffffffff] reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: BIOS-e820: [mem 0x0000000100000000-0x000000021f37ffff] usable
Mai 17 14:07:15 dorian-ms7a38 kernel: NX (Execute Disable) protection: active
Mai 17 14:07:15 dorian-ms7a38 kernel: SMBIOS 3.0 present.
Mai 17 14:07:15 dorian-ms7a38 kernel: DMI: Micro-Star International Co., Ltd. MS-7A38/B350M PRO-VDH (MS-7A38), BIOS A.80 12/05/2017
Mai 17 14:07:15 dorian-ms7a38 kernel: tsc: Fast TSC calibration failed
Mai 17 14:07:15 dorian-ms7a38 kernel: e820: update [mem 0x00000000-0x00000fff] usable ==> reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: e820: remove [mem 0x000a0000-0x000fffff] usable
Mai 17 14:07:15 dorian-ms7a38 kernel: last_pfn = 0x21f380 max_arch_pfn = 0x400000000
Mai 17 14:07:15 dorian-ms7a38 kernel: x86/PAT: Configuration [0-7]: WB  WC  UC- UC  WB  WP  UC- WT
Mai 17 14:07:15 dorian-ms7a38 kernel: e820: update [mem 0xe0000000-0xffffffff] usable ==> reserved
Mai 17 14:07:15 dorian-ms7a38 kernel: last_pfn = 0xdf000 max_arch_pfn = 0x400000000
Mai 17 14:07:15 dorian-ms7a38 kernel: Using GB pages for direct mapping
Mai 17 14:07:15 dorian-ms7a38 kernel: RAMDISK: [mem 0x3286b000-0x3542cfff]
Mai 17 14:07:15 dorian-ms7a38 kernel: ACPI: Early table checksum verification disabled
